dev/bots/test.dart

```diff
@@ -802,13 +808,17 @@ Future<void> _runFlutterTest(String workingDirectory, {
   String script,
   bool expectFailure = false,
   bool printOutput = true,
+  OutputChecker outputChecker,
   List<String> options = const <String>[],
   bool skip = false,
   Duration timeout = _kLongTimeout,
   bq.TabledataResourceApi tableData,
   Map<String, String> environment,
   List<String> tests = const <String>[],
 }) async {
+  // Support printing output or capturing it for matching, but not both.
```

Comment:
Something like this should be the message on the assert rather than a comment, e.g. `assert(!printOutput || outputChecker == null, 'Output can either be captured or matched, but not both')`

Reply:
Oops.  Fixed.